pkg/resources/prometheusRules_test.go

```diff
@@ -100,9 +100,9 @@ func TestReconcileAlerts(t *testing.T) {
 			},
 			Installation: &integreatlyv1alpha1.RHMI{
 				ObjectMeta: v1.ObjectMeta{
-					Name:              "rhoam",
-					Namespace:         "testing-namespaces-test",
-					DeletionTimestamp: now(),
+					Name:      "rhoam",
+					Namespace: "testing-namespaces-test",
+					//DeletionTimestamp: now(),
```

Comment:
why is this commented out?

Reply:
I think it's not critical for test to manage deletion grace period; the default for Kubelet to delete the object is 30 seconds . Hope it's ok for test.  
It's also to avoid Error in Unit test - `panic: failed to add runtime object... fake client: refusing to create obj  with metadata.deletionTimestamp but no finalizers`

Reply:
If it's not required, please remove instead of commenting out.